helm/chart-operator/values.yaml

```diff
@@ -45,6 +45,7 @@ resource:
     name: '{{ .Release.Name | replace "." "-" | trunc 47 }}-psp'
 
 tiller:
+  image: "giantswarm/tiller:v2.16.8"
```

Comment:
Later when we would like to update a tiller after this security incident, we can simple override this value in default-catalog configmap.